template/apps/web/src/services/api.service.ts

```diff
@@ -1,14 +1,21 @@
-// eslint-disable-next-line max-classes-per-file
 import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
 
 import config from 'config';
 
+type ApiErrorHandler = (error: ApiError) => void;
+
+type EventHandler = ApiErrorHandler;
```

Comment:
if another handler should be added, developer should update `EventHandler` type adding own handler type